torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -57,16 +57,24 @@ namespace fs = std::experimental::filesystem;
 
 #ifndef _WIN32
 #include <sys/stat.h>
+#include <sys/types.h> 
+#include <limits.h>
+#include <unistd.h>
 #endif
 
 // HACK for failed builds in ARVR, where it cannot find these symbols within
 // std::experimental::filesystem
 namespace {
-fs::path get_current_path() {
-#if __has_include("filesystem")
-  return fs::current_path();
+std::string get_current_path() {
+#ifdef _WIN32
```

Comment:
I would change it to `#if __has_include("filesystem") && !defined(__LINUX__)` as we want to use those on both Windows and Mac, where we don't have this problem